src/Compilers/Core/Portable/Compilation/SymbolInfo.cs

```diff
@@ -10,51 +10,33 @@
 
 namespace Microsoft.CodeAnalysis
 {
-    public struct SymbolInfo : IEquatable<SymbolInfo>
+    public readonly struct SymbolInfo : IEquatable<SymbolInfo>
     {
-        internal static readonly SymbolInfo None = new SymbolInfo(null, ImmutableArray<ISymbol>.Empty, CandidateReason.None);
+        internal static readonly SymbolInfo None = default;
```

Comment:
there was previously an observable distinction between the 'default' and 'None' instances, if you looked very closely.  But it was also very strange as pairwise property comparisons would not show any difference.  I've simplified this such that the `default` version is the same as `new SymbolInfo()` which is the same as `new SymbolInfo(default)/new SymbolInfo(default, default),new SymbolInfo(default, default, default)`.

Reply:
I think this behavior change needs a look from compat council. Could you send an email?

Reply:
@333fred actually, this is an internal property.  Even if this was passed externally, anyone observing any of it's properties would see teh same values as before.  e.g. `.CandidateSymbols` will be a non-empty array in both cases.  

Do you think this still needs compat review?

Reply:
The behavior of `default(SymbolInfo)` has changed. I think this deserves an email.